### ISSUE TITLE
SQLite file must be moved to a specific directory

### ISSUE THREAD

**Issue by sanacl** [chaoss/grimoirelab-hatstall#48]
Currently the sqlite file is stored in the same directory with source code (scripts). This makes difficult to split logic (source code) from data (database) when the software it is used via docker containers. 

If you try to use a docker volume for the sqlite file, the git clone get crashed:
```
fatal: destination path 'grimoirelab-hatstall' already exists and is not an empty directory.`
```

The setup is available at: https://github.com/chaoss/grimoirelab-hatstall/blob/master/django-hatstall/django_hatstall/settings.py#L81

The simple solution for this would be using a path like `/data` or `/db` for hosting the sqlite file. If you can be docker agnostic then this should be a parameter somewhere and by default it should never be stored in the same directory source code lives (a subdirectory such as `~/.hatstall/db` would do the trick)

**Comment by sanacl** [comment]
I've been having a look at this and it is complex with the current Dockefile + stage. If we use a docker environment variable it will be used by the `grimoirelab` user who installs the hatstall code, but that user is not the one who execute it. So the environment variable does not work unless we compose a configuration file with it.

The other way is to define a fixed path for the sqlite file, which is not sexy at all.

**Comment by sanacl** [comment]
Fixed by #50
